Add a note about teams as flag maintainers - Expected pub date 2022-12-05 (#310)
launchdarkly/LaunchDarkly-Docs: src/content/topics/home/using-flags/flag-settings.mdx

The flag maintainer is the account member who is primarily responsible for the flag. By default, the maintainer is set to the member who created the flag, but you can assign any member of your team as the maintainer for a particular flag.

Enterprise customers using [teams](/home/teams) can assign a team as the flag's maintainer. This offers the additional benefit of shared accountability and also ensures that if one member leaves the team, the flag still has other maintainers.

> The flag's maintainer can be updated on the flag's **Settings** tab, or from the header of any of the flag's tabs:
> 
> ![Clicking the pencil icon in the header of the flag page allows you to set a new flag maintainer.](edit-flag-maintainer.png)

Reviewer: ```suggestion
You can update the flag's maintainer on the flag's **Settings** tab. You can also update the maintainer by clicking the pencil icon from the header of any of the flag's tabs:

![The flag's maintainer and "pencil" icon.](edit-flag-maintainer.png)
```

Style nits -- active voice, and our captions [describe what the reader sees](https://launchdarkly.atlassian.net/wiki/spaces/PDH/pages/342786398/Style+guide+Tech+Writing+Documentation#Screenshot-captions), rather than the action they might do from here.

Author: Ah, thank you!

The flag maintainer is the account member who is primarily responsible for the flag. By default, the maintainer is set to the member who created the flag, but you can assign any member of your team as the maintainer for a particular flag.

> Enterprise customers using [teams](/home/teams) can assign a team as the flag's maintainer. This offers the additional benefit of shared accountability and also ensures that when one member leaves the team, the flag still has other owners.

Reviewer: ```suggestion
Enterprise customers using [teams](/home/teams) can assign a team as the flag's maintainer. This offers the additional benefit of shared accountability and also ensures that if one member leaves the team, the flag still has other maintainers.
```

I think it's fine to add a screenshot here if you'd like one @charliepark, as long as you crop it fairly closely. It's not a problem to add a screenshot that's for Enterprise, although we may want to mention that in the caption in addition to here in preceding paragraph. 

Does the `flag-settings-all-environments.png` (line 43) need to be updated based on this feature?

Author: The image currently on the page isn't wrong … it's just showing a single member as flag maintainer, which is still a valid state that a flag could be in (either enterprise or not). So we can leave that one as-is, I think.
